content/Contributing/UITranslations.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "String Translations"
+description: "How to contribute CORE text string translations."
```

Comment:
Is this article meant to be applicable for both CORE and SCALE? If so, then I have a small suggestion.
```suggestion
description: "How to contribute TrueNAS interface text string translations."
```